### DISCUSSION TITLE
Specifying a subdir when triggering a DAG via REST API

### DISCUSSION THREAD

**Discussion by nivfteradata** [apache/airflow#22032] [General]
### Description

Nowadays, it seems that it is not possible to list/trigger DAGs using a subdir parameter via REST API, although it is supported on the Airflow CLI commands

### Use case/motivation

The relevant list/trigger DAGs REST API operations to support the subdir parameter

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment]
Why do you need this parameter? The webserver does not read DAG files, but only uses serialized DAGs from the database, so this parameter doesn't make sense.

**Comment by potiuk** [comment]
Converting to discussion until more information provided.
